uint16 red_1_robot_HP
uint16 red_2_robot_HP
uint16 red_3_robot_HP
uint16 red_4_robot_HP
uint16 red_5_robot_HP
uint16 red_7_robot_HP
uint16 red_outpost_HP
uint16 red_base_HP
uint16 blue_1_robot_HP
uint16 blue_2_robot_HP
uint16 blue_3_robot_HP
uint16 blue_4_robot_HP
uint16 blue_5_robot_HP
uint16 blue_7_robot_HP
uint16 blue_outpost_HP
uint16 blue_base_HP